### PR TITLE
Added location corrodinates for SonnyTate

### PR DESCRIPTION
**Pull request by sonnytate**
This pull request closes life, the universe, and everything. #42

### CODE REVIEW

File: _pins/SonnyTate [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
```
Comment (sonnytate) [inline _pins/SonnyTate]: To Ford Perfect: Get Sonny off from this planet before the Vogon constructor fleet arrives. Give him this message: "Don't panic". Oh don't forget to get Arthur too...

Comment (sonnytate) [inline _pins/SonnyTate]: Hey Ford, I'm still waiting...

File: _pins/SonnyTate [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ githubHandle: Sonny
+ latitude: 36.769272
+ longitude: -76.184563
```
Comment (ljur) [inline _pins/SonnyTate]: @sonnytate try removing blank spaces in front of each row

**Comment by crichID** [comment]
@sonnytate you are so close! All you need to do is add the `.json` file extension to your file and all of the tests should pass! Please let me know if you need any help.

**Comment by crichID** [comment]
@sonnytate I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3034/commits/5cb0857363fdb61804203d79e2cedef8e044871d. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**